### DISCUSSION TITLE
`minimum-contrast` makes black font white on white background

### DISCUSSION THREAD

**Discussion by jonashaag** [ghostty-org/ghostty#4174] [Q&A]
```
theme = dark:iTerm2 Dark Background,light:iTerm2 Light Background
minimum-contrast = 1
```
<img width="488" alt="image" src="https://github.com/user-attachments/assets/04754070-1ae1-4fce-aad1-344cb77aa411" />

If you increase `minimum-contrast`:

```
theme = dark:iTerm2 Dark Background,light:iTerm2 Light Background
minimum-contrast = 1.1
```
<img width="488" alt="image" src="https://github.com/user-attachments/assets/4f7f2666-9214-4f79-b7de-69034bad3f93" />

Either I don't understand how `minimum-contrast` is supposed to work or there's a bug.

**Comment by rhodes-b** [comment]
https://github.com/ghostty-org/ghostty/discussions/3869
if you havent seen it

**Reply by jonashaag** [reply]
I don't understand how that's related

**Reply by rhodes-b** [reply]
its the same issue except in the theme viewer please read the thread

**Reply by jonashaag** [reply]
I did read it. Black on white has maximum contrast already so why should black be set to white, actually reducing contrast?

**Reply by rhodes-b** [reply]
https://github.com/ghostty-org/ghostty/discussions/3869#discussioncomment-11692911

https://github.com/mbadolato/iTerm2-Color-Schemes/blob/master/wayst/iTerm2%20Light%20Background#L1-L2

Its related to these two lines

**Reply by jonashaag** [reply]
Thank for trying to help. Unfortunately I cannot follow your very "sparse"  explanations. Maybe if you'd provide a full chain of thoughts and facts, I could follow along more easily.

**Reply by rhodes-b** [reply]
works just fine for me either way, you need to share more of your setup or its macos specific

![image](https://github.com/user-attachments/assets/4f4bbbf3-d4d5-4e02-9f74-5c960bf3c704)
![image](https://github.com/user-attachments/assets/a338ce6d-516b-49b8-9460-e03834ba68f7)

**Reply by jonashaag** [reply]
<img width="522" alt="image" src="https://github.com/user-attachments/assets/4777e64a-31af-479c-befd-392f46276d9e" />
<img width="416" alt="image" src="https://github.com/user-attachments/assets/6b3e0a42-0d89-4ac5-9d3f-bc242314d00d" />

seems macOS specific

**Reply by jonashaag** [reply]
> works just fine for me either way

does this mean your explanations were incorrect after all?
